content/en/docs/tasks/security/authentication/authn-policy/index.md

```diff
@@ -62,270 +62,119 @@ sleep.legacy to httpbin.bar: 200
 sleep.legacy to httpbin.legacy: 200
 {{< /text >}}
 
-You should also verify that there is a default mesh authentication policy in the system, which you can do as follows:
+Verify there is no peer authentication policy in the system with the following command:
 
 {{< text bash >}}
-$ kubectl get policies.authentication.istio.io --all-namespaces
+$ kubectl get peerauthentication --all-namespaces
 No resources found.
 {{< /text >}}
 
-{{< text bash >}}
-$ kubectl get meshpolicies.authentication.istio.io
-NAME      AGE
-default   3m
-{{< /text >}}
-
 Last but not least, verify that there are no destination rules that apply on the example services. You can do this by checking the `host:` value of
  existing destination rules and make sure they do not match. For example:
 
 {{< text bash >}}
 $ kubectl get destinationrules.networking.istio.io --all-namespaces -o yaml | grep "host:"
-    host: istio-policy.istio-system.svc.cluster.local
-    host: istio-telemetry.istio-system.svc.cluster.local
 {{< /text >}}
 
 {{< tip >}}
 Depending on the version of Istio, you may see destination rules for hosts other then those shown. However, there should be none with hosts in the `foo`,
 `bar` and `legacy` namespace, nor is the match-all wildcard `*`
 {{< /tip >}}
 
-## Globally enabling Istio mutual TLS
+## Auto mTLS
 
-To set a mesh-wide authentication policy that enables mutual TLS, submit *mesh authentication policy* like below:
+By default, Istio tracks the server workloads migrated to Istio proxies, and configures client proxies to send mutual TLS traffic to those workloads automatically, and to send plain text traffic to workloads without sidecars.
```

Comment:
we use `Istio proxies` in the first sentence and `without sidecars` in the latter one. The rest also use `proxies/sideacar`, suggest to stick to one of them consistently

Reply:
(perfer) sidecar.

Reply:
unless for a good reason, I suggest to use sidecar. reasons:

https://istio.io/docs/reference/glossary/ usage of sidecar is more common, proxy e.g. can be referred to egress gateway, https://istio.io/docs/reference/glossary/#tls-origination

doing search in istio.io website give me similar correlation

https://istio.io/search/?q=sidecar
https://istio.io/search/?q=proxy

Reply:
We have tried to move away from using sidecar in favor of proxies. The reason is that while all sidecars have proxies, not all proxies are in sidecars. If I missed to replace an instance of "sidecar" with "proxy" please make the change.
As you point out, there are still a lot of mentions of sidecars that must be scrutinized to ensure they are accurate.
This is a longer conversation that I'm happy to continue post release.